client/blocks/post-likes/docs/example.jsx

```diff
@@ -43,11 +39,7 @@ class PostLikesExample extends React.PureComponent {
 		} );
 	};
 
-	setPopoverContext = element => {
-		this.setState( {
-			popoverContext: element,
-		} );
-	};
+	setPopoverContext = ref => ( this.popoverContext = ref );
```

Comment:
React 16.3 has a very nice new helper for refs like this:
```jsx
class X {
  popoverContext = React.createRef();

  render() {
    return (
      <div>
        <button ref={ this.popoverContext } />
        <Popover context={ this.popoverContext.current } />
      </div>
    );
  }
}
```

And setting state when setting a ref is certainly a bug: many mysterious infinite loops must have been caused by this. I witnessed many with popover components, but was never able to reproduce and debug.

Reply:
do you want me to update to `React.createRef()` in this PR?

Reply:
Yes, please update to `React.createRef()` if you don't mind. If we don't do it now, there will be very little motivation to do it at some point in future.

Reply:
Updated in 8cd49fa0b1